plugin.video.iplayerwww/default.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import division
+
+import os
+import sys
+import urllib
+
+import xbmc
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+
+
+plugin_handle = int(sys.argv[1])
+ADDON = xbmcaddon.Addon(id='plugin.video.iplayerwww')
+sys.path.insert(0, os.path.join(ADDON.getAddonInfo("path"), 'resources', 'lib'))
+
+try:
+    import ipwww_common as Common
+    from ipwww_common import utf8_unquote_plus, CreateBaseDirectory, KidsMode
+    import ipwww_video as Video
+    import ipwww_radio as Radio
+except ImportError as error:
+    d = xbmcgui.Dialog()
+    d.ok(str(error), 'Please check you installed this plugin correctly.')
```

Comment:
Any message shown in the interface has to be localized

Reply:
Should be fixed.